scene/2d/sprite_2d.cpp

```diff
@@ -329,6 +343,30 @@ int Sprite2D::get_hframes() const {
 	return hframes;
 }
 
+void Sprite2D::set_separation(const Vector2i &p_separation) {
+	if (separation == p_separation) {
+		return;
+	}
+	if (p_separation.width >= 0) {
+		separation.width = p_separation.width;
+	} else {
+		separation.width = 0;
+	}
+
+	if (p_separation.height >= 0) {
+		separation.height = p_separation.height;
+	} else {
+		separation.height = 0;
+	}
```

Comment:
```suggestion
	
	separation = Vector2i(0, 0).max(p_separation);
```
Equivalent and compact, also clearer IMO

Reply:
That it is, thanks!